src/main/webapp/css/main.css

```diff
@@ -16,18 +16,104 @@
 
 /* Prevent the page from becoming too wide. */
 body {
+	background: #5433FF;  /* fallback for old browsers */
+  background: -webkit-linear-gradient(to right, #A5FECB, #20BDFF, #5433FF);  /* Chrome 10-25, Safari 5.1-6 */
+  background: linear-gradient(to right, #A5FECB, #20BDFF, #5433FF); /* W3C, IE 10+/ Edge, Firefox 16+, Chrome 26+, Opera 12+, Safari 7+ */
+  color: #828c9b;
+	font-family: Product Sans;
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
   max-width: 600px;
+  width: 50%;
+
+}
+
+/* Creation of dropdown menu. */
+.dropbtn {
+  background-color: #828c9b;
+  border: none;
+ 	color: white;
+ 	cursor: pointer;	
+  font-size: 16px;
+	padding: 16px;
+  position: absolute;
+  right: 0;
+}
+
+.dropdown {
+  display: inline-block;
+  position: absolute;
+  right: 20px;
+}
+
+.dropdown-content {
+	background-color: #f9f9f9;
+	box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
+	display: none;
+  min-width: 160px;
+  position: relative;
+  top: 32px;
+  z-index: 1;
+}
+
+.dropdown-content a {
+  color: black;
+  display: block;
+  padding: 12px 16px;
+  text-decoration: none;
+}
+
+.dropdown-content a:hover {
+background-color: #f1f1f1
+}
+
+.dropdown:hover .dropdown-content {
+  display: block;
+}
+
+.dropdown:hover .dropbtn {
+  background-color: #3e8e41;
+}
+
+/*@font-face {
```

Comment:
why is this commented out

Reply:
Couldnt get the font to pass the stickler test, Its a font I wanted to use more, but I need help on syntax.